.eslintignore

```diff
@@ -1,2 +1,5 @@
 /backstop_data
 /dist
+/src/index.html
+/src/styles/style.css
```

Comment:
we shouldn't ignore these files, fix all mistakes you have on check

Reply:
Sorry, could you describe in more detailed what should I do?